CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # bedrock-vc-delivery ChangeLog
 
+## 3.3.0 - 2023-07-TBD
+
+### Changed
+- Make `expectedCredentialRequests` optional in `openIdExchangeOptions` schema.
+
+### Added
```

Comment:
`Added` should go before `Changed` (alphabetize).

Reply:
Updated [b9a1202](https://github.com/digitalbazaar/bedrock-vc-delivery/pull/30/commits/b9a120235cb0f65d05b29fbae8dd6c016a7abf1f)